src/test-helpers/e2e-helpers/mobile/paste.ts

```diff
@@ -0,0 +1,14 @@
+import { Browser } from 'webdriverio'
+import execute from './execute'
+
+/** Import text on given unranked path using exposed testHelpers. */
+const paste = async (browser: Browser<'async'>, unrankedPath: string[], text: string) => {
```

Comment:
Does `webdriverio` have a way to trigger a native `paste` event so that we don't have to use the hack that we needed for puppeteer?

Reply:
No problem. If webdriverio doesn't support it then we probably need to inject it like we are doing. What do you mean by separate util function?

Reply:
Webdriver io doesn't support paste events but I think we can combine some events to achieve paste events.
For example, Long press the editable and then find a native element that contains 'paste' text and then touch this native element, etc.  I am not sure if it will work but I will give it a shot at a more convenient time.